### ISSUE TITLE
Test commands don't execute when running tests on IE11.

### ISSUE THREAD

**Issue by BadNews87** [webdriverio/webdriverio#6780]
[//]: # NOTE: This repository only maintains packages that are listed in the [Readme](https://github.com/webdriverio/webdriverio/blob/main/README.md#packages). Please make sure that your issue is directly caused by one of these packages and if not file an issue in the correct 3rd party package repository.

**Environment (please complete the following information):**
 - **WebdriverIO version:** 7.5.2
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 14.15.4
 - **NPM version:** 6.14.10
 - **Browser name and version:**  Internet Explorer 11
 - **Platform name and version:** Windows 10
 - **Additional wdio packages used (if applicable):** ``` "@wdio/allure-reporter": "^7.5.2",
    "@wdio/browserstack-service": "^7.5.2",
    "@wdio/cli": "^7.5.2",
    "@wdio/local-runner": "^7.5.2",
    "@wdio/mocha-framework": "^7.5.2",
    "@wdio/selenium-standalone-service": "^7.5.2",
    "@wdio/spec-reporter": "^7.5.2",
    "@wdio/sync": "^7.5.2",```

**Config of WebdriverIO**
**wdio.shared.conf.js**
```
exports.config = {
    /*
     *
     * ====================
     * Runner Configuration
     * ====================
     *
     * WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
     * on a remote machine).
     */
    runner: 'local',
    /*
     * ==================
     * Specify Test Files
     * ==================
     * Define which test specs should run. The pattern is relative to the directory
     * from which `wdio` was called. Notice that, if you are calling `wdio` from an
     * NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
     * directory is where your package.json resides, so `wdio` will be called from there.
     *
     */
    specs: [
        './test/specs/**/example*.js',
    ],
    // Patterns to exclude.
    exclude: [
        // './test/specs/**/example.e2e.js'
        // 'path/to/excluded/files'
    ],
    /*
     *
     * ============
     * Capabilities
     * ============
     * Define your capabilities here. WebdriverIO can run multiple capabilities at the same
     * time. Depending on the number of capabilities, WebdriverIO launches several test
     * sessions. Within your capabilities you can overwrite the spec and exclude options in
     * order to group specific specs to a specific capability.
     *
     * First, you can define how many instances should be started at the same time. Let's
     * say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
     * set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
     * files and you set maxInstances to 10, all spec files will get tested at the same time
     * and 30 processes will get spawned. The property handles how many capabilities
     * from the same test should run tests.
     *
     */
    maxInstances: 10,
    /*
     *
     * ===================
     * Test Configurations
     * ===================
     * Define all options that are relevant for the WebdriverIO instance here
     *
     * Level of logging verbosity: trace | debug | info | warn | error | silent
     */
    logLevel: 'warn',
    /*
     *
     * Set specific log levels per logger
     * loggers:
     * - webdriver, webdriverio
     * - @wdio/applitools-service, @wdio/browserstack-service,
     *  @wdio/devtools-service, @wdio/sauce-service
     * - @wdio/mocha-framework, @wdio/jasmine-framework
     * - @wdio/local-runner
     * - @wdio/sumologic-reporter
     * - @wdio/cli, @wdio/config, @wdio/sync, @wdio/utils
     * Level of logging verbosity: trace | debug | info | warn | error | silent
     * logLevels: {
     *     webdriver: 'info',
     *     '@wdio/applitools-service': 'info'
     * },
     *
     * If you only want to run your tests until a specific amount of tests have failed use
     * bail (default is 0 - don't bail, run all tests).
     */
    bail: 0,
    /*
     *
     * Set a base URL in order to shorten url command calls. If your `url` parameter starts
     * with `/`, the base url gets prepended, not including the path portion of your baseUrl.
     * If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
     * gets prepended directly.
     */
    baseUrl: 'http://localhost',
    /*
     *
     * Default timeout for all waitFor* commands.
     */
    waitforTimeout: 10000,
    /*
     *
     * Default timeout in milliseconds for request
     * if browser driver or grid doesn't send response
     */
    connectionRetryTimeout: 120000,
    /*
     *
     * Default request retries count
     */
    connectionRetryCount: 3,
    /*
     *
     * Test runner services
     * Services take over a specific job you don't want to take care of. They enhance
     * your test setup with almost no effort. Unlike plugins, they don't add new
     * commands. Instead, they hook themselves up into the test process.
     */

    /*
     * Framework you want to run your specs with.
     * The following are supported: Mocha, Jasmine, and Cucumber
     * see also: https://webdriver.io/docs/frameworks
     *
     * Make sure you have the wdio adapter package for the specific framework installed
     * before running any tests.
     */
    framework: 'mocha',
    /*
     *
     * The number of times to retry the entire specfile when it fails as a whole
     * specFileRetries: 1,
     *
     * Delay in seconds between the spec file retry attempts
     * specFileRetriesDelay: 0,
     *
     * Whether or not retried specfiles should be retried immediately
     *  or deferred to the end of the queue
     * specFileRetriesDeferred: false,
     *
     * Test reporter for stdout.
     * The only one supported by default is 'dot'
     * see also: https://webdriver.io/docs/dot-reporter
     */
    reporters: ['spec', ['allure', { outputDir: 'allure-results', disableWebdriverScreenshotsReporting: false }]],

    /*
     *
     * Options to be passed to Mocha.
     * See the full list at http://mochajs.org/
     */
    mochaOpts: {
        // Babel setup
        require: ['@babel/register'],
        ui: 'bdd',
        timeout: 60000,
    },
    /*
     *
     * =====
     * Hooks
     * =====
     * WebdriverIO provides several hooks you can use to interfere
     *  with the test process in order to enhance
     * it and to build services around it. You can either apply a single function or an array of
     * methods to it. If one of them returns with a promise,
     *  WebdriverIO will wait until that promise got
     * resolved to continue.
     */
    /**
     * Gets executed once before all workers get launched.
     * @param {Object} config wdio configuration object
     * @param {Array.<Object>} capabilities list of capabilities details
     */
    /*
     * onPrepare: function (config, capabilities) {
     * },
     */
    /**
     * Gets executed before a worker process is spawned
     *  and can be used to initialise specific service
     * for that worker as well as modify runtime environments in an async fashion.
     * @param  {String} cid      capability id (e.g 0-0)
     * @param  {[type]} caps     object containing capabilities for session
     *                           that will be spawn in the worker
     * @param  {[type]} specs    specs to be run in the worker process
     * @param  {[type]} args     object that will be merged with the main configuration
     *                           once worker is initialised
     * @param  {[type]} execArgv list of string arguments passed to the worker process
     */
    /*
     * onWorkerStart: function (cid, caps, specs, args, execArgv) {
     * },
     */
    /**
     * Gets executed just before initialising the webdriver session and test framework.
     * It allows you
     * to manipulate configurations depending on the capability or spec.
     * @param {Object} config wdio configuration object
     * @param {Array.<Object>} capabilities list of capabilities details
     * @param {Array.<String>} specs List of spec file paths that are to be run
     */
    /*
     * beforeSession: function (config, capabilities, specs) {
     * },
     */
    /**
     * Gets executed before test execution begins. At this point you can access to all global
     * variables like `browser`. It is the perfect place to define custom commands.
     * @param {Array.<Object>} capabilities list of capabilities details
     * @param {Array.<String>} specs        List of spec file paths that are to be run
     * @param {Object}         browser      instance of created browser/device session
     */
    /*
     * before: function (capabilities, specs) {
     * },
     */
    /**
     * Runs before a WebdriverIO command gets executed.
     * @param {String} commandName hook command name
     * @param {Array} args arguments that command would receive
     */
    /*
     * beforeCommand: function (commandName, args) {
     * },
     */
    /**
     * Hook that gets executed before the suite starts
     * @param {Object} suite suite details
     */
    /*
     * beforeSuite: function (suite) {
     * },
     */
    /**
     * Function to be executed before a test (in Mocha/Jasmine) starts.
     */
    /*
     * beforeTest: function (test, context) {
     * },
     */
    /**
     * Hook that gets executed _before_ a hook within the suite starts (e.g. runs before calling
     * beforeEach in Mocha)
     */
    /*
     * beforeHook: function (test, context) {
     * },
     */
    /**
     * Hook that gets executed _after_ a hook within the suite starts (e.g. runs after calling
     * afterEach in Mocha)
     */
    /*
     * afterHook: function (test, context, { error, result, duration, passed, retries }) {
     * },
     */
    /**
     * Function to be executed after a test (in Mocha/Jasmine).
     */
    afterTest (test, context, {
 // eslint-disable-next-line no-unused-vars
 error, result, duration, passed, retries,
}) {
        if (!passed) {
            browser.takeScreenshot();
        }
    },

    /**
     * Hook that gets executed after the suite has ended
     * @param {Object} suite suite details
     */
    /*
     * afterSuite: function (suite) {
     * },
     */
    /**
     * Runs after a WebdriverIO command gets executed
     * @param {String} commandName hook command name
     * @param {Array} args arguments that command would receive
     * @param {Number} result 0 - command success, 1 - command error
     * @param {Object} error error object if any
     */
    /*
     * afterCommand: function (commandName, args, result, error) {
     * },
     */
    /**
     * Gets executed after all tests are done. You still have access to all global variables from
     * the test.
     * @param {Number} result 0 - test pass, 1 - test fail
     * @param {Array.<Object>} capabilities list of capabilities details
     * @param {Array.<String>} specs List of spec file paths that ran
     */
    /*
     * after: function (result, capabilities, specs) {
     * },
     */
    /**
     * Gets executed right after terminating the webdriver session.
     * @param {Object} config wdio configuration object
     * @param {Array.<Object>} capabilities list of capabilities details
     * @param {Array.<String>} specs List of spec file paths that ran
     */
    /*
     * afterSession: function (config, capabilities, specs) {
     * },
     */
    /**
     * Gets executed after all workers got shut down and the process is about to exit. An error
     * thrown in the onComplete hook will result in the test run failing.
     * @param {Object} exitCode 0 - success, 1 - fail
     * @param {Object} config wdio configuration object
     * @param {Array.<Object>} capabilities list of capabilities details
     * @param {<Object>} results object containing test results
     */
    /*
     * onComplete: function(exitCode, config, capabilities, results) {
     * },
     */
    /**
     * Gets executed when a refresh happens.
     * @param {String} oldSessionId session ID of the old session
     * @param {String} newSessionId session ID of the new session
     */
    /*
     * onReload: function(oldSessionId, newSessionId) {
     * }
     */
};
```

**wdio.local.conf.js**
```
const { config } = require('./wdio.shared.conf');

exports.config = {

    ...config,

    ...{
        capabilities: [{

            /*
             * maxInstances can get overwritten per capability. So if you have an in-house Selenium
             * grid with only 5 firefox instances available you can make sure that not more than
             * 5 instances get started at a time.
             */
            maxInstances: 5,
            //
            browserName: 'internet explorer',
						"se:ieOptions": {
							ignoreZoomSetting: true
						}
            /*
             * If outputDir is provided WebdriverIO can capture driver session logs
             * it is possible to configure which logTypes to include/exclude.
             * excludeDriverLogs: ['*'], // pass '*' to exclude all driver session logs
             * excludeDriverLogs: ['bugreport', 'server'],
             */
        }],
        services: ['selenium-standalone'],
				path: '/wd/hub',
    },

};
```

**Describe the bug**
Commands do not execute on IE 11. I always get a timeout. They work on other browsers.

**To Reproduce**
1. Setup the project.
2. Create any test.
3. Execute the test.

You can clone my repo & just change the **_wdio.local.conf.js_**  to execute on IE 11.

https://github.com/BadNews87/wdio-testing.git

**Expected behavior**
The tests should execute. Commands like click & send keys should work.

**Log**
If applicable, add logs to help explain your problem.
[gist](https://gist.github.com/BadNews87/d13b59ec9c2c532041a1ecf99bf4293d)

**Additional context**
Was working some time ago. Approximately 1 month.

**Comment by BadNews87** [comment]
My project is actually ```"@wdio/cli": "^7.0.3",``` - I tried to update to the newest version 7 it didn't help.

**Comment by christian-bromann** [comment]
@BadNews87 it seems that your test fails with

```
Error in "My Login application should login with valid credentials"
Error: Timeout of 60000ms exceeded. The execution in the test "My Login application should login with valid credentials" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts).
```

which caused the other test to fail as well. Can you try to increase the timeout and retry?

**Comment by christian-bromann** [comment]
> Timeout of 100000ms exceeded

Seems that 100s is not enough.

**Comment by christian-bromann** [comment]
Closing due to inactivity. Please check out [the timeout docs](https://webdriver.io/docs/timeouts) for more information. There are different timeouts for different things.
